cg-spec/editors_draft.html

```diff
@@ -2301,8 +2302,9 @@ <h3>sparql-compare</h3>
           error, based on the comparison of two
           <a href="#dfn-rdf-star-terms">RDF-star terms</a>.
           When comparing two RDF-star triple terms, comparison is 
-          by pairwise comparison of subject, then predicate, then object.
-        </p>
+          by pairwise comparison of components subject, then predicate, then object.
```

Comment:
```suggestion
          by pairwise comparison of their components: first of their subjects; then of their predicates; then of their objects.
```

Reply:
I'll reword this text. The details are left ot the definition box (e.g. only compare predicates if the subjects are the same).